src/ProposalPage.js

```diff
@@ -99,6 +99,7 @@ export default class ProposalPage extends React.Component {
           {showVoteControls &&
             <VoteControls
               proposal={proposal}
+              proposalId={proposal.id}
```

Comment:
Why?

Reply:
Cause I need numerical representation of proposal in that original array. As a lot of code and contracts depend on it.

Reply:
I mean you can use this id without passing as props: you have a whole proposal object

Reply:
Oh! 😮
You are right! I will fix it. Thank you @sunify!